src/main/java/com/rht_labs/Project.java

```diff
@@ -45,4 +58,54 @@ public String openEndpoint(@Context SecurityContext ctx) {
     public String securedEndpoint(@Context SecurityContext ctx) {
         return jwt.getName();
     }
+
+    @GET
+    @Path("git")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String readFromRepo(@Context SecurityContext ctx) throws Exception {
+        // Clone repo
+        File workingDir = Files.createTempDirectory("workspace").toFile();
+        TransportConfigCallback transportConfigCallback = new SshTransportConfigCallback();
+        Git git = Git.cloneRepository()
+                .setDirectory(workingDir)
+                .setTransportConfigCallback(transportConfigCallback)
+                .setURI(System.getenv("GIT_REPO_URL"))
+                .call();
+
+        // Create a branch
+        git.branchCreate().setName("omp-new-branch").call();
```

Comment:
will this fail if the repo already has that branch? Assuming poc I'm fine with it.